pom.xml

```diff
@@ -95,8 +95,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.11.0</version>
                 <configuration>
-                    <source>1.8</source>
```

Comment:
As stated already, this needs to be reverted. Java 1.8 is our current target.

Reply:
I'll made the necessary changes here thanks.